### PR TITLE
Pretty donations in transaction view

### PR DESCRIPTION
**Pull request by tmb**

### CODE REVIEW

**Comment by tmb** [comment]
Second commit is because `calculate_fee_amount` wasn't being run when fee reimbursements were manually matched, leading to some really weird math being done while trying to calculate the fee.

File: app/views/donations/_donation.html.erb [hunk 1]
```diff
@@ -0,0 +1,75 @@
+<% transaction = donation.payout.t_transaction %>
```
Comment (thesephist) [inline app/views/donations/_donation.html.erb]: The indentation on this template is kind of fucked can we fix it

File: app/controllers/transactions_controller.rb [hunk 2]
```diff
@@ -98,6 +99,14 @@ def update
         should_delete_fee_relationship = true if fee_relationship&.persisted?
       end
 
+      if current_fee_reimbursement.nil? && @transaction.fee_reimbursement.present?
+        @transaction.fee_relationship = FeeRelationship.new(
+          event_id: @transaction.fee_reimbursement.event.id,
+          fee_applies: true,
+          fee_amount: @transaction.fee_reimbursement.calculate_fee_amount
+        )
+      end
```
Comment (thesephist) [inline app/controllers/transactions_controller.rb]: It seems weird that this was missing before but it worked for invoices -- if it's being handled here now for all transactions, should we remove the code that was only handling it for invoices before, so we only have one place in the code that concerns this?

**Review by thesephist — CHANGES_REQUESTED** [review]
need a lil fix 📈

File: app/controllers/transactions_controller.rb (hunk 2 above)
Comment (tmb) [reply app/controllers/transactions_controller.rb]: Invoices don't have the weird under $1 fee stuff—that's what thsi fixes

**Pull request merged**